package.json

```diff
@@ -7,7 +7,8 @@
   "license": "MIT",
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "turbo run turbobuild --single-package",
+    "turbobuild": "next build",
```

Comment:
Something funky about single package mode (at least right now) is that if I call `turbo run build` here it will recursively call itself, so `npm run build` needs to point to `turbo run <something_else>`.